pom.xml

```
@@ -141,7 +141,7 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>docker-workflow</artifactId>
-        <version>1.14</version>
+        <version>1.22-rc452.c6ab7e23b1c4</version> <!-- TODO https://github.com/jenkinsci/docker-workflow-plugin/pull/202 -->
```

Must be released first!

---

1.22 was just released:

```suggestion
        <version>1.22</version>
```